libass.yaml

```
@@ -29,19 +29,38 @@ pipeline:
       tag: ${{package.version}}
 
   - uses: meson/configure
+    with:
+      opts: -Ddefault_library=both
 
   - uses: meson/compile
 
   - uses: meson/install
 
   - uses: strip
 
+subpackages:
+  - name: ${{package.name}}-dev
+    description: Development files for libass
+    pipeline:
+      - uses: split/dev
+    test:
+      pipeline:
+        - name: Binary reliability checks
+          uses: test/ldd-check
+          with:
+            packages: $(basename ${{subpkg.name}})
```

```suggestion
          uses: test/tw/ldd-check
```
this by default takes the context name into account.